app/views/offenders/show.html.haml

```diff
@@ -8,16 +8,19 @@
     %span.app-typography--body-1= @offender[:sid]
   .col.m5
     .app-typography--caption-above Offender Name
-    %span.app-typography--body-1= @offender[:name]
-.row.hide-on-med-and-down
-  .col.m5.offset-m1
-    .app-typography--caption-above Status
-    %span.app-typography--body-1= @offender[:status]
-  .col.m5
-    .app-typography--caption-above Offender Photo
     %span.app-typography--body-1
-      = link_to 'Show Photo', "http://docpub.state.or.us/OOS/imageLoader.jsp?idno=#{@offender[:sid]}", target: "_blank"
+      = @offender.include?(:name) ? @offender[:name] : "#{@offender[:first]} #{@offender[:last]}"
 .row.hide-on-med-and-down
+  - if @offender[:status]
+    .col.m5.offset-m1
+      .app-typography--caption-above Status
+      %span.app-typography--body-1= @offender[:status]
+  - if @offender[:jurisdiction] == :oregon
```

Comment:
Using an arbitrary variable like this concerns me a bit; if you accidentally committed a typo like `== :orgon` this would fail silently. You could either write a test for that case, or create a class constant that you refer to from everywhere and that'll error if you misspell it. I'm not sure what the accepted approach for that is in ruby/rails.

I'm not sure what the accepted  are in Rails, but using an arbitrary variable like this